fe1-web/__tests__/model/objects/WalletCryptographyHandler.test.ts

```diff
@@ -1,51 +1,11 @@
 import 'jest-extended';
 import { WalletCryptographyHandler } from 'model/objects/WalletCryptographyHandler';
 import { sign } from 'tweetnacl';
-import STRINGS from 'res/strings';
-import { get } from 'idb-keyval';
+import { WalletCryptoLibrary } from 'res/adhoc-implementations/__mocks__/WalletCryptoLibrary';
```

Comment:
you should just be able to call the jest mocking function, as we had discussed.
no need to pass this as a parameter or reference the mock explicitly, I believe.